flake.nix

```diff
@@ -2,7 +2,42 @@
   # WARNING this is very much still experimental.
   description = "A collection of darwin modules";
 
-  outputs = { self, nixpkgs }: {
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, flake-utils, nixpkgs }:
+    let
+      darwinSystems =
+        builtins.filter
+          (sys: builtins.isList (builtins.match "^[[:alnum:]_]+-darwin$" sys))
```

Comment:
Might be simpler to use `hasSuffix` here

Reply:
Alternatively, you could hardcode it to `aarch64-darwin` and `x86_64-darwin`, as currently it also includes `armv7a-darwin` and `i686-darwin`